Source/Core/CMakeLists.txt

```diff
@@ -2,7 +2,10 @@ add_subdirectory(AudioCommon)
 add_subdirectory(Common)
 add_subdirectory(Core)
 add_subdirectory(DiscIO)
-add_subdirectory(DolphinWX)
+if(NOT DISABLE_WX)
+  add_subdirectory(DolphinWX)
```

Comment:
@vladfi1 Can you give some more context on this change? Isn't this already handled implicitly by not calling `find_package(wxWidgets)`?